qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -80,7 +80,9 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
             circ = qiskit.QuantumCircuit(1, 1)
             circ.h(0)
             circ.delay(delay, 0, self._unit)
-            circ.p(2 * np.pi * self._osc_freq, 0)
+            nosc = delay / self._delays[0]
```

Comment:
Could `_delays[0]` be 0? if so this could raise an error

Reply:
1. Added a check that `delays` are strictly increasing.
2. If delay[0] is 0 then take delay[1] as min value for division to avoid the error you pointed out.